packages/lib-classifier/src/plugins/tasks/DrawingTask/models/DrawingTask.spec.js

```diff
@@ -107,6 +107,12 @@ describe('Model > DrawingTask', function () {
       drawingTask.setActiveMark(point1)
       expect(drawingTask.activeMark).to.equal(point1)
     })
+
+    it('should hide previous marks', function () {
+      drawingTask.togglePreviousMarks()
```

Comment:
Could you add expectations before the action is called so we can be sure it is doing what we expect?
```suggestion
      expect(drawingTask.hidePreviousMarks).to.be.false()
      expect(drawingTask.hidingIndex).to.be.null()
      drawingTask.togglePreviousMarks()
```